app/views/error.nunjucks

```diff
@@ -1,13 +1,13 @@
 {% extends 'layout.nunjucks' %}
 
 {% block header %}
-Ooops...an error has occured.
+A server error has occured.
 {% endblock %}
 
 {% block content %}
-<div onclick="(function myFunction() { var error = document.getElementById('error-message'); error.style.display = error.style.display == 'none' ? 'block' : 'none'; })()">
-Looks like one of the techies has pulled a sickie. Please try again or get in touch if it keeps happening.
-<div id="error-message" style="display:none;">
+<p>This error page should only appear in non-production envirnoments.</p>
```

Comment:
Removing the error message would be a good idea.

I understand the desire to be able to immediately see if there is a problem with the config using this page but I think we should probably be using other things to test that. If you really wanted to do that, perhaps just have something in the source of the page e.g. `generated by app` as the differentiator?

I'll hold off merging until the page has been stripped back.